packages/ag-charts-website/src/content/docs/api-state/_examples/state-save-restore/main.ts

```diff
@@ -13,21 +7,11 @@ import {
     CrosshairModule,
     ZoomModule,
 } from 'ag-charts-enterprise';
-import { FinancialChartModule, OrdinalTimeAxisModule } from 'ag-charts-enterprise';
+import { FinancialChartModule } from 'ag-charts-enterprise';
 
 import { getData } from './data';
 
-ModuleRegistry.registerModules([
-    AnimationModule,
-    CrosshairModule,
-    FinancialChartModule,
-    LegendModule,
-    NumberAxisModule,
-    OrdinalTimeAxisModule,
-    TimeAxisModule,
-    UnitTimeAxisModule,
-    ZoomModule,
-]);
+ModuleRegistry.registerModules([AnimationModule, CrosshairModule, FinancialChartModule, LegendModule, ZoomModule]);
```

Comment:
`FinancialChartModule` already registers `AnimationModule`, `CrosshairModule` and `ZoomModule`